.github/workflows/main-build.yml

```diff
@@ -17,8 +17,8 @@ jobs:
         # TODO_TECHDEBT: upgrade to the latest Ignite (the latest at the moment of creating a note is 0.28). Need to downgrade to fix CI pipelines. Might be done in scope of #240.
         run: |
           # curl https://get.ignite.com/cli! | bash
-          wget https://github.com/ignite/cli/releases/download/v0.27.2/ignite_0.27.2_linux_amd64.tar.gz
-          tar -xzf ignite_0.27.2_linux_amd64.tar.gz
+          wget https://github.com/ignite/cli/releases/download/v28.2.0/ignite_28.2.0_linux_amd64.tar.gz
+          tar -xzf ignite_28.2.0_linux_amd64.tar.gz
```

Comment:
Shouldn't we use `v28.1.0` this seems to be what rollkit are recommending in their docs

https://rollkit.dev/tutorials/gm-world#install-ignite-cli-linux

Reply:
@red-0ne `28.2.0` was just released yesterday, I'm sure Rollkit team just didn't have an opportunity to update the docs. The new version seems to work fine.